Source/WebCore/platform/graphics/ImageBackingStore.h

```diff
@@ -124,38 +122,36 @@ class ImageBackingStore {
         if (rect.isEmpty() || !inBounds(rect))
             return;
 
-        size_t rowBytes = rect.width() * sizeof(uint32_t);
-        uint32_t* src = pixelAt(rect.x(), rect.y());
-        uint32_t* dest = src + m_size.width();
+        auto src = pixelAt(rect.x(), rect.y());
+        auto dest = src.subspan(m_size.width());
         for (int i = 1; i < rect.height(); ++i) {
-            memcpy(dest, src, rowBytes);
-            dest += m_size.width();
+            memcpySpan(dest, src.first(rect.width()));
+            dest = dest.subspan(m_size.width());
         }
     }
 
-    uint32_t* pixelAt(int x, int y) const
+    std::span<uint32_t> pixelAt(int x, int y) const
```

Comment:
This function name made more sense when it returned only a pointer. But I don't suppose I have a better name. Maybe with the word _after_ in it?

Reply:
Proposing `pixelsStartingAt()` which returns a `std::span<uint32_t>` and `pixelAt()` which returns a `uint32_t&`